src/views/Senate.js

```diff
@@ -185,15 +163,15 @@ export default function Senate() {
         </>
       );
     });
-  }, [open, polls, votingOn]);
+  }, [open, polls, votingOn, doVote]);
 
   const onDocumentChange = useCallback(({ valid, values }) => {
```

Comment:
why comment these instead of removing?

Reply:
I agree it's not very tidy, but I imagine we'll want to get the "verify/create a document hash" functionality back in once we upgrade this to a more fully-featured senate page.

@vvisigoth we _are_ going to upgrade this to a more fully-featured senate page sometime next year, right?